modules/social_features/social_activity/social_activity.module

```diff
@@ -62,7 +43,30 @@ function social_activity_social_user_account_header_items(array $context) {
         '#wrapper_attributes' => ['class' => 'divider'],
         '#markup' => '',
       ],
-      'notifications' => $notifications_view,
+      // Element to contain the link for ajaxified view for notifications.
+      'load_link' => [
```

Comment:
We should create a follow-up to change the throbber here to our spinner that's currently the contents of `notifications`. After this we can clear the markup of the `notifications` element. 

This can be a drupal.org issue for Open Social : )

Reply:
Created: https://www.drupal.org/project/social/issues/3074740